network/acceptor_test.go

```diff
@@ -291,12 +304,17 @@ func (suite *AcceptorSuite) TestAcceptorSuccess() {
 	rep := &ReputationManagerMock{}
 	rep.On("Success", address)
 
+	subscriber := make(chan interface{}, 1)
+
 	// act
-	handleAccepting(suite.log, &suite.wg, &suite.cfg, pending, peers, rep, conn)
+	handleAccepting(suite.log, &suite.wg, &suite.cfg, pending, peers, rep, conn, subscriber)
 
 	// assert
 	pending.AssertCalled(suite.T(), "Claim", address)
 	pending.AssertCalled(suite.T(), "Release", address)
 	peers.AssertCalled(suite.T(), "Add", conn, nonce)
 	rep.AssertCalled(suite.T(), "Success", address)
+	event := <-subscriber
```

Comment:
This will block if there is no event on the channel. You should try to find a better way.